src/plugins/guided_onboarding/public/components/guide_panel.tsx

```diff
@@ -59,52 +60,82 @@ const getProgress = (state?: GuideState): number => {
   return 0;
 };
 
+const errorSection = (
+  <EuiFlyoutBody>
+    <EuiEmptyPrompt
```

Comment:
What do you think about keeping the `EuiFlyoutHeader` with the "Back to guides" link? That might also fix the small overlap here with the close icon.

<img width="588" alt="Screen Shot 2022-12-14 at 2 25 12 PM" src="https://user-images.githubusercontent.com/5226211/207706394-d8afa79a-af2b-4dec-b7b4-41f1f488aa19.png">

Reply:
Great suggestion, @alisonelizabeth! I ended up putting the "back to guides" button in the flyout body together with the error prompt to avoid weird padding issues. But it definitely looks better with it.